src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/CcdKeyValue.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CcdKeyValue {
+
+    public final String key;
+    public final String value;
+
+    public CcdKeyValue(@JsonProperty("key") String key, @JsonProperty("value") String value) {
+        this.key = key;
+        this.value = value;
+    }
```

Comment:
Do you need this constructor? The request body will be serialised from public fields assigned to this object.
Looking at the code below seems like more convenient way to construct it is from actual ocr map entry e.g.

```java
public CcdKeyValue(Map.Entry<String, String> entry) {
    this.key = entry.getKey();
    this.value = entry.getValue();
}
```

Reply:
I think it's clearer and in the long run easier to maintain when a data class receives its fields in the constructor and doesn't have any conversion logic, especially when the constructor is as simple as this one.

Reply:
`CcdKeyValue` type is essentially `Map.Entry` anyway. Just thought it makes life easier if things are alike etc. Well, suggested along with the mapping function really 😉.
The question whether you need constructor was incomplete. I think we don't need `@JsonProperty` annotations there for sure

Reply:
You're right about `@JsonProperty`. Well spotted, I don't know what I was thinking.